### PR TITLE
adding `oras manifest fetch` to the docs "Pushing_and_pulling"

### LINKED ISSUE #229: [Docs] - since docs were written before developing `oras manifest fetch` , it hasn't been discussed at required places
### Code of Conduct

- [X] I have read and agree to the ORAS' [Code of Conduct](https://github.com/oras-project/oras-www/blob/main/CODE_OF_CONDUCT.md)

### What part of document/web-page on oras.land is affected?

1. [pushing and pulling guide](https://oras.land/docs/how_to_guides/pushing_and_pulling) 

issue11

it says "The push would generate the following manifest", the problem is how to output the manifest should also be specified. we do it through 
```
oras manifest fetch localhost:5000/hello-artifact:v2 --pretty
```
2. [manifest annotation](https://oras.land/docs/how_to_guides/manifest_annotations)

issu12

again it says "results in" but we must specify how to output the manifest , this is done through -
```
oras manifest fetch localhost:5000/club:party --pretty
```
@qweeah was kind enough to answer my question and told me that  `oras manifest fetch` was developed after the creation of docs, I think the docs should be updated likewise.

### Additional comments?

_No response_

### PR DESCRIPTION
**Pull request by 1Shubham7**
Fixes #229 

Screenshots
image

### CODE REVIEW

File: docs/how_to_guides/manifest_annotations.mdx [hunk 1]
```diff
@@ -82,6 +82,12 @@ In order to add annotations using the `annotations.json`, you may run the follow
 oras push --annotation-file annotations.json localhost:5000/club:party cake.txt juice.txt
 ```
 
+This creates a manifest file. Running the following command to output the manifest:
```
Comment (sajayantony) [inline docs/how_to_guides/manifest_annotations.mdx:85]: Maybe word this to align with the APIs. Also the file might not be  generated locally and an in memory implementation of the registry would not generate a file. So would not say file in either place. 

The push command would upload the blob and put a manifest. 
Running the …

File: docs/how_to_guides/pushing_and_pulling.mdx [hunk 2]
```diff
@@ -97,7 +97,13 @@ See [OCI Artifacts](https://github.com/opencontainers/image-spec/blob/main/manif
     ./docs/:application/vnd.acme.rocket.docs.layer.v1+tar
   ```
 
-- The push would generate the following manifest:
+- The push will generate a manifest file. To output it, run the following command:
```
Comment (sajayantony) [inline docs/how_to_guides/pushing_and_pulling.mdx:100]: Same comment here about removing the concept of file.

**Review by sajayantony — DISMISSED** [review]

File: docs/how_to_guides/manifest_annotations.mdx (hunk 1 above)
Comment (1Shubham7) [reply docs/how_to_guides/manifest_annotations.mdx]: Ok @sajayantony, got it.

**Comment by 1Shubham7** [comment]
Done @sajayantony, Sorry I misspelled your name in the commits.

**Review by qweeah — APPROVED** [review]
Thanks @1Shubham7. LGTM although IANAM

**Comment by 1Shubham7** [comment]
@sajayantony Please review it, I think its good now

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Review by asmitbm — APPROVED** [review]
LGTM 🚀

**Pull request merged**